apps/emqx_dashboard/src/emqx_dashboard.erl

```diff
@@ -310,6 +313,24 @@ api_key_authorize(Req, HandlerInfo, Key, Secret) ->
             )
     end.
 
+jwt_token_bearer_authorize(Req, HandlerInfo, Token) ->
+    case emqx_dashboard_admin:verify_token(Req, HandlerInfo, Token) of
+        {ok, #{actor := Username} = ActorContext} ->
+            AuthnMeta = #{
```

Comment:
AuthnMeta deserve a type spec for it ? too imporant that `namespace` cannot be escaping.

what is the meaning of `<< >>`  namespace? could this be used to escape?